CrabScreenSaverView.swift

```diff
@@ -49,14 +79,35 @@ class CrabScreenSaverView: ScreenSaverView {
         crabs[0].xPosition += 1
         crabs[1].xPosition -= 1
         
+        // REVIEW: It's a little confusing how you have 2 crabs hardcoded, but SOME of
+        // the logic is written for N-many crabs, but some assumes only 2 crabs...
```

Comment:
I get the dichotomy. Looping this way saves me from having to keep a magic number someplace like this is C.